components/ipfs/ipfs_ports.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ipfs/ipfs_ports.h"
+
+#include "base/strings/string_number_conversions.h"
+
+namespace {
+
+// Default ports used for non-official builds.
```

Comment:
hmm, where do we check for official vs non-official?.. Aren't these used for any builds?

Reply:
Comment updated.